book/src/internal/build.md

```diff
@@ -17,14 +17,13 @@ The process of building a CXX-Qt project internally is complex and contains mult
 ## 1. Cargo build
 
 When cargo build is triggered, this causes dependencies to be downloaded and built. One of these is
-`cxx-qt-lib` which has helper Qt types. It contains a `build.rs` which uses CXX to generate C++
-sources for the Qt types. These are stored into the `OUT_DIR` of the crate, then as the `cxx-qt-lib`
-is built it uses `include_str` to expose the generates files as `const` strings in the crate.
+`cxx-qt-lib` which defines helper Qt types. It contains a `build.rs` which uses CXX to generate C++
+sources for the Qt types. These are stored into the `OUT_DIR` of the crate.
+Note that we cannot know where the `OUT_DIR` or other files of `cxx-qt-lib` are, from the main
```

Comment:
Think this either needs an extra comma or one less comma? (first seems better to me)
Note that we cannot know where the `OUT_DIR` or other files of `cxx-qt-lib` are, from the main project, as they are in hashed directories.

Or
Note that we cannot know where the `OUT_DIR` or other files of `cxx-qt-lib` are from the main project as they are in hashed directories.

Reply:
Done